server/mlabns/util/lookup_query.py

```diff
@@ -193,8 +197,17 @@ def _get_user_defined_lat_lon(self, request):
 
     def _set_maxmind_geolocation(self, ip_address, country, city):
         geo_record = maxmind.GeoRecord()
+        address_family = None
+
+        if _is_valid_ipv6(ip_address):
+            address_family = message.ADDRESS_FAMILY_IPv6
+        elif _is_valid_ipv4(ip_address):
+            address_family = message.ADDRESS_FAMILY_IPv4
+
         if ip_address is not None:
-            geo_record = maxmind.get_ip_geolocation(ip_address)
+            logging.debug('Getting maxmind info for ip %s in family %s',
+                          ip_address, address_family)
+            geo_record = maxmind.get_ip_geolocation(ip_address, address_family)
```

Comment:
Do we need to do the v4/v6 logic in this function and pass the results in a parameter? Seems like it would be more simple to do it in `get_ip_geolocation` -- are you doing this because you already have `_is_valid_ipv6` here?